src/Psl/ObjectOriented/class_alias.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\ObjectOriented;
```

Comment:
`Object` and `Class` are reserved, `Obj` and `Cls` are "weird", and `OO` is weirder.

i think `ObjectOriented` makes the most sense.

Reply:
Projects Like PhpParser solve this by adding a `_`, e.g. `Object_`. But i think this works fine too

Reply:
Couldn't come up with a better name either...

Reply:
`Class` could be used as a namespace in PHP 8.0, i think i'm pulling a @Ocramius card here and requiring PHP ^8.0 in PSL 1.7 🤔

Reply:
I can remember the pain of having classes named after a type whilst upgrading from php 5 to 7 and would not recommend that. Even though you can in php 8.

My IDE will autocomplete the namespace, so I don't mind that it is named object-oriented. That much.

But it's up to you :)

Reply:
> My IDE will autocomplete the namespace, so I don't mind that it is named object-oriented. That much.

personally i don't import symbols, but rather namespaces most of the time.

so it's:

```php
use Psl\Class;

if (Class\exists($foo)) {
 ...
}
```

rather than:

```php
use function Psl\Class\exists;

if (exists($foo)) {
 ...
}
```

and here `Class\exists($foo)` reads better than `ObjectOriented\class_exists($foo)` ( and definitly better than `exists($foo)` ).